geps/gep-1897.md

```diff
@@ -0,0 +1,172 @@
+collecting evidence that this feature is still unresolved.  This was followed by
+[GEP: Describe Backend Properties #1282](https://github.com/kubernetes-sigs/gateway-api/issues/1282).
+
+In August 2022, [Add Provisional GEP-1282 document #1333](https://github.com/kubernetes-sigs/gateway-api/pull/1333)
+was created, and in October 2022, a GEP update with proposed implementation
+[GEP-1282 Backend Properties - Update implementation #1430](https://github.com/kubernetes-sigs/gateway-api/pull/1430)
+was followed by intense discussion and closed in favor of a downsize in scope.
+
+In January 2023 we closed GEP-1282 and began a new discussion on enumerating TLS use cases in
+[Gateway API TLS Use Cases](#references), for the purposes of a clear definition and separation of concerns.
+This GEP is the outcome of the TLS use cases #4 and #5 in
+[Gateway API TLS Use Cases](#references) as mentioned in the Goals section above.
+
+## API
+
+Details deferred until we reach consensus on what we want to do, and why we want to do this.
+
+## Prior Art
+
+TLS from gateway to backend for ingress exists in several implementations, and was developed independently.
+
+### Istio Gateway supports this with a DestinationRule to encrypt the gateway->backend hop:
+
+* A secret representing a certificate/key pair, where the certificate is valid for the route host
+* Set Gateway spec.servers[].port.protocol: HTTPS, spec.servers[].tls.mode=SIMPLE, spec.servers[].tls.credentialName:
+* Set DestinationRule spec.trafficPolicy.tls.mode: SIMPLE
+
+Ref: [Istio / Understanding TLS Configuration](https://istio.io/latest/docs/ops/configuration/traffic-management/tls-configuration/#gateways)
+and [Istio / Destination Rule](https://istio.io/latest/docs/reference/config/networking/destination-rule/#ClientTLSSettings)
+
+### OpenShift Route (comparable to GW API Gateway) supports this with the following route configuration items:
+
+* A certificate/key pair, where the certificate is valid for the route host
+* A separate destination CA certificate enables the Ingress Controller to trust the destination’s certificate
+* An optional, separate CA certificate that completes the certificate chain
+
+Ref: [Secured routes - Configuring Routes | Networking | OpenShift Container Platform 4.12](https://docs.openshift.com/container-platform/4.12/networking/routes/secured-routes.html#nw-ingress-creating-a-reencrypt-route-with-a-custom-certificate_secured-routes)
+
+### Contour supports this from Envoy to the backend using:
+
+* An Envoy client certificate
+* A CA certificate and SubjectName which are both used to verify the backend endpoint’s identity.
+* Kubernetes Service annotation: projectcontour.io/upstream-protocol.tls
+
+Ref: [Upstream TLS](https://projectcontour.io/docs/v1.21.1/config/upstream-tls/)
+
+### GKE supports a way to encrypt traffic to the backend pods using:
+
+* `AppProtocol` on Service set to HTTPS
+* Certificate verification by pod
```

Comment:
Not sure what this means but I don't think GKE verifies anything

Reply:
Updated